go/tasks/pluginmachinery/utils/dns.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"regexp"
+	"strings"
+)
+
+var dns1123MaxLen = 63
```

Comment:
There is already available as DNS1123LabelMaxLength
which is  part api machinery validation function and can be utilized instead of this

Reply:
Fixed